Bug Crash While Running

Travis Bot crashed with this [error](https://github.com/ray-project/travis-tracker-v2/runs/2135752127?check_suite_focus=true):
```
Traceback (most recent call last):
  File "fetch_and_render.py", line 437, in <module>
    db.write_travis_data(travis_data)
  File "fetch_and_render.py", line 257, in write_travis_data
    status = TRAVIS_TO_BAZEL_STATUS_MAP[travis_commit.state]
KeyError: 'queued'
make: *** [Makefile:6: data] Error 1
```

--- a/fetch_and_render.py
+++ b/fetch_and_render.py
@@ -143,6 +143,7 @@ def yield_test_result(bazel_log_path):
 
 TRAVIS_TO_BAZEL_STATUS_MAP = {
     "created": None,
+    "queued": None,
     "errored": "FAILED",
     "failed": "FAILED",
     "passed": "PASSED",

--- a/fetch_and_render.py
+++ b/fetch_and_render.py
@@ -254,7 +254,7 @@ class ResultsDB:
                         f"SELECT * FROM test_result WHERE job_id == {travis_job_id}"
                     ).fetchall()
                 )
-                status = TRAVIS_TO_BAZEL_STATUS_MAP[travis_commit.state]
+                status = TRAVIS_TO_BAZEL_STATUS_MAP.get(travis_commit.state)
                 if num_result == 0 and status is not None:
                     records_to_insert.append(
                         (

